diff --git a/node_modules/reka-ui/dist/Menu/MenuContentImpl.cjs b/node_modules/reka-ui/dist/Menu/MenuContentImpl.cjs
index 4998def..143b427 100644
--- a/node_modules/reka-ui/dist/Menu/MenuContentImpl.cjs
+++ b/node_modules/reka-ui/dist/Menu/MenuContentImpl.cjs
@@ -155,7 +155,7 @@ var MenuContentImpl_vue_vue_type_script_setup_true_lang_default = /* @__PURE__ *
 				arrowKeyOptions: "vertical",
 				dir: rootContext?.dir.value,
 				focus: false,
-				attributeName: "[data-reka-collection-item]:not([data-disabled])"
+				attributeName: "[data-reka-collection-item]"
 			});
 			if (el) {
 				highlightedElement.value = el;
@@ -202,7 +202,7 @@ var MenuContentImpl_vue_vue_type_script_setup_true_lang_default = /* @__PURE__ *
 				arrowKeyOptions: "vertical",
 				dir: rootContext?.dir.value,
 				focus: true,
-				attributeName: "[data-reka-collection-item]:not([data-disabled])"
+				attributeName: "[data-reka-collection-item]"
 			});
 			if (el) return el?.focus();
 			if (event.code === "Space") return;
diff --git a/node_modules/reka-ui/dist/Menu/MenuContentImpl.js b/node_modules/reka-ui/dist/Menu/MenuContentImpl.js
index 4543a04..e6dbf67 100644
--- a/node_modules/reka-ui/dist/Menu/MenuContentImpl.js
+++ b/node_modules/reka-ui/dist/Menu/MenuContentImpl.js
@@ -154,7 +154,7 @@ var MenuContentImpl_vue_vue_type_script_setup_true_lang_default = /* @__PURE__ *
 				arrowKeyOptions: "vertical",
 				dir: rootContext?.dir.value,
 				focus: false,
-				attributeName: "[data-reka-collection-item]:not([data-disabled])"
+				attributeName: "[data-reka-collection-item]"
 			});
 			if (el) {
 				highlightedElement.value = el;
@@ -201,7 +201,7 @@ var MenuContentImpl_vue_vue_type_script_setup_true_lang_default = /* @__PURE__ *
 				arrowKeyOptions: "vertical",
 				dir: rootContext?.dir.value,
 				focus: true,
-				attributeName: "[data-reka-collection-item]:not([data-disabled])"
+				attributeName: "[data-reka-collection-item]"
 			});
 			if (el) return el?.focus();
 			if (event.code === "Space") return;
