diff --git a/node_modules/svelte-sonner/dist/Toast.svelte b/node_modules/svelte-sonner/dist/Toast.svelte
index a15126a..006952a 100644
--- a/node_modules/svelte-sonner/dist/Toast.svelte
+++ b/node_modules/svelte-sonner/dist/Toast.svelte
@@ -519,7 +519,7 @@
 
 <!-- svelte-ignore a11y_no_noninteractive_tabindex -->
 <li
-	tabindex={0}
+	tabindex={-1}
 	bind:this={toastRef}
 	class={cn(
 		restProps.class,
@@ -529,8 +529,6 @@
 		classes?.[toastType],
 		toast?.classes?.[toastType]
 	)}
-	aria-live={toast.important ? 'assertive' : 'polite'}
-	aria-atomic="true"
 	data-sonner-toast=""
 	data-rich-colors={toast.richColors ?? defaultRichColors}
 	data-styled={!(toast.component || toast.unstyled || unstyled)}
diff --git a/node_modules/svelte-sonner/dist/Toaster.svelte b/node_modules/svelte-sonner/dist/Toaster.svelte
index 51385c9..29987f0 100644
--- a/node_modules/svelte-sonner/dist/Toaster.svelte
+++ b/node_modules/svelte-sonner/dist/Toaster.svelte
@@ -243,7 +243,7 @@
 
 	onMount(() => {
 		const handleKeydown = (event: KeyboardEvent) => {
-			const isHotkeyPressed = hotkey.every(
+			const isHotkeyPressed = hotkey.length > 0 && hotkey.every(
 				(key) =>
 					// eslint-disable-next-line @typescript-eslint/no-explicit-any
 					(event as any)[key] || event.code === key
